CactbotOverlay/FFXIVProcessIntl.cs

```diff
@@ -0,0 +1,465 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Newtonsoft.Json.Linq;
+
+namespace Cactbot {
+  public class FFXIVProcessIntl : FFXIVProcess {
+
+    [StructLayout(LayoutKind.Explicit)]
+    public unsafe new struct EntityMemory {
+      public static int Size => Marshal.SizeOf(typeof(EntityMemory));
+
+      // Unknown size, but this is the bytes up to the next field.
+      public const int nameBytes = 68;
+
+      [FieldOffset(0x30)]
+      public fixed byte Name[nameBytes];
+
+      [FieldOffset(0x74)]
+      public uint id;
+
+      [FieldOffset(0x8C)]
+      public EntityType type;
+
+      [FieldOffset(0x92)]
+      public ushort distance;
+
+      [FieldOffset(0xA0)]
+      public Single pos_x;
+
+      [FieldOffset(0xA4)]
+      public Single pos_z;
+
+      [FieldOffset(0xA8)]
+      public Single pos_y;
+
+      [FieldOffset(0xB0)]
+      public Single rotation;
+
+      [FieldOffset(0x18B8)]
+      public CharacterDetails charDetails;
+    }
+
+    [StructLayout(LayoutKind.Explicit)]
+    public new struct CharacterDetails {
+
+      [FieldOffset(0x00)]
+      public int hp;
+
+      [FieldOffset(0x04)]
+      public int max_hp;
+
+      [FieldOffset(0x08)]
+      public int mp;
+
+      [FieldOffset(0x0C)]
+      public int max_mp;
+
+      [FieldOffset(0x12)]
+      public short gp;
+
+      [FieldOffset(0x14)]
+      public short max_gp;
+
+      [FieldOffset(0x16)]
+      public short cp;
+
+      [FieldOffset(0x18)]
+      public short max_cp;
+
+      [FieldOffset(0x3C)]
+      public EntityJob job;
+
+      [FieldOffset(0x3E)]
+      public short level;
+
+      [FieldOffset(0x5F)]
+      public short shieldPercentage;
+    }
+    public FFXIVProcessIntl(ILogger logger) : base(logger) { }
+
+    [Serializable]
```

Comment:
FFXIVProcess.cs has dummy entries for all of the JobMemory structs and I've defined them in each version as although they're identical at the moment a future patch could change them and it saves work splitting them later on.